**Linux with security manager** [Build 3372](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=75556&buildTypeId=WF_PullRequest_LinuxSm) outcome was **FAILURE** using a merge of b6fda08a5b208473d388ddd4f5746b0b2dd00ae6
Summary: Tests failed: 1 (1 new), passed: 956, ignored: 293 Build time: 00:07:20
### Failed tests
```
org.jboss.as.clustering.jgroups.ManagedSocketFactoryTest.createDatagram: java.lang.AssertionError: expected same:<Mock for DatagramSocket, hashCode: 16236629> was not:<null>
	at org.jboss.as.clustering.jgroups.ManagedSocketFactoryTest.createDatagram(ManagedSocketFactoryTest.java:135)


```

--- a/clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/ManagedSocketFactory.java
+++ b/clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/ManagedSocketFactory.java
@@ -99,7 +99,8 @@ public class ManagedSocketFactory implements SocketFactory {
 
     @Override
     public DatagramSocket createDatagramSocket(String name, SocketAddress address) throws SocketException {
-        return this.manager.createDatagramSocket(name, address);
+        // Creating the socket using the name throws an ISE, See WFCORE-1063
+        return this.manager.createDatagramSocket(address);
     }
 
     @Override

--- a/clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/ManagedSocketFactoryTest.java
+++ b/clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/ManagedSocketFactoryTest.java
@@ -113,7 +113,7 @@ public class ManagedSocketFactoryTest {
     }
 
     @Test
-    public void createDatagram() throws IOException {
+    public void createDatagramSocket() throws IOException {
 
         DatagramSocket socket1 = mock(DatagramSocket.class);
         DatagramSocket socket2 = mock(DatagramSocket.class);
@@ -122,10 +122,10 @@ public class ManagedSocketFactoryTest {
         InetAddress localhost = InetAddress.getLocalHost();
         SocketAddress socketAddress = new InetSocketAddress(localhost, 2);
 
-        when(this.manager.createDatagramSocket("test", new InetSocketAddress(0))).thenReturn(socket1);
-        when(this.manager.createDatagramSocket("test", new InetSocketAddress(1))).thenReturn(socket2);
-        when(this.manager.createDatagramSocket("test", socketAddress)).thenReturn(socket3);
-        when(this.manager.createDatagramSocket("test", new InetSocketAddress(localhost, 1))).thenReturn(socket4);
+        when(this.manager.createDatagramSocket(new InetSocketAddress(0))).thenReturn(socket1);
+        when(this.manager.createDatagramSocket(new InetSocketAddress(1))).thenReturn(socket2);
+        when(this.manager.createDatagramSocket(socketAddress)).thenReturn(socket3);
+        when(this.manager.createDatagramSocket(new InetSocketAddress(localhost, 1))).thenReturn(socket4);
 
         DatagramSocket result1 = this.subject.createDatagramSocket("test");
         DatagramSocket result2 = this.subject.createDatagramSocket("test", 1);

